api/integrations/basecamp/zulip_basecamp_mirror

```diff
@@ -120,7 +121,7 @@ def run_mirror():
         since = since.string
     except (AssertionError, IOError) as e:
         logging.warn("Could not open resume file: %s" % (e.message or e.strerror,))
-        since = (datetime.utcnow() - timedelta(hours=config.BASECAMP_INITIAL_HISTORY_HOURS)).isoformat() + "-00:00"
+        since = (timezone.now() - timedelta(hours=config.BASECAMP_INITIAL_HISTORY_HOURS)).isoformat() + "-00:00"
```

Comment:
We can't use Django code inside `api/integrations`, because that code is distributed separately from the Zulip server.  

So for these, we'll need to use some other solution.